x-pack/plugin/core/src/main/resources/logs-json-pipeline.json

```diff
@@ -0,0 +1,48 @@
+{
+  "processors": [
+    {
+      "rename": {
+        "if": "ctx?.message?.startsWith('{') == true && ctx?.message?.endsWith('}') == true",
```

Comment:
> Nit: I think you can remove the ? from ctx?., as the context object is guaranteed to be non-null.

I thought the same, but saw it is used all over 🙈 

> I think this condition would fail if message is not a string...

With https://github.com/elastic/elasticsearch/issues/95538, it shouldn't be possible, but I guess no harm in adding such test and validation

> It's also not clear to my why == true was needed to fix some tests here.

The `?.` operator returns `null`, so I thought failures are related to using it in a boolean phrase with `&&`, but I couldn't actually reproduce this failure with a `null` message